### PR TITLE
[MO][TF FE] Recover MOC TF FE unit-tests in the pre-commit

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Recover MOC TF FE unit-tests in the pre-commit

**Ticket:** 109220

### CODE REVIEW

**Review by mvafin — APPROVED** [review]

File: .ci/azure/linux.yml [hunk 1]
```diff
@@ -326,6 +326,7 @@ jobs:
 
   - script: |
       export LD_LIBRARY_PATH=$(REPO_DIR)/temp/gna_03.05.00.1906/linux/x64:$(LD_LIBRARY_PATH)
+      export PYTHONPATH=$(INSTALL_TEST_DIR):$(INSTALL_DIR)/python/python3.8:$PYTHONPATH
```
Comment (rkazants) [inline .ci/azure/linux.yml:329]: Needed to import openvino.test_util to get IR comparator

**Pull request merged**